dashboard/app/views/levels/editors/_applab.html.haml

```diff
@@ -104,3 +104,13 @@
   = render partial: 'levels/editors/checkboxes', locals: {f: f, field_name: :debugger_disabled, description: "Disable code debugger (Console is still enabled)"}
 
 = render partial: 'levels/editors/teacher_only_markdown', locals: {f: f}
+
+.field
+  = f.label :helper_libraries
+  %p
+    Select
+    %a.select_all{href: '#'} all
+    \/
+    %a.select_none{href: '#'} none
+    (shift-click or cmd-click to select multiple).
+  = f.select :helper_libraries, (Library.distinct.pluck(:name) + (@level.helper_libraries || [])).uniq.sort, {}, {multiple: true}
```

Comment:
Can this field be extracted to a partial instead of duplicated across the App Lab and Game Lab editors?

Reply:
I'm not super worried about this b/c I think we'll likely move this field up higher (into a shared view) in the future.  I'd imagine more level types will eventually have Helper Library code.